test/unit/user_event/type_test.dart

```diff
@@ -197,12 +199,32 @@ void _typeTestHelper({bool hasDelay = false, bool isTextArea = false}) {
       });
     });
 
+  });
+
+  group('', () {
     testEventHandlerErrors(
       ['onInput'],
       (el) => hasDelay ? UserEvent.typeWithDelay(el, 'K', Duration(hours: 500)) : UserEvent.type(el, 'K'),
       isTextArea ? react.textarea : react.input,
     );
-  });
+
+    test('multiple event handler errors on every letter', () {
+      const stringToTest = 'Hello There';
+      final view = rtl.render((isTextArea ? react.textarea : react.input)({
+        defaultTestIdKey: 'event-handle-error-tester',
+        'onKeyUp': (e) => throw ExceptionForTesting('ow'),
```

Comment:
Can we add some comment here saying that this is `keyUp` since it happens after the delay, and won't throw synchronously?

Reply:
Sorry, I didn't read this test properly 😓 ... disregard this comment, sorry about that!